icons/activities.svg

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Now that we have different kinds of resources. Its better to collect them under a folder.
So please create a folder `resource`. And move `fonts` and `icons` inside.